### DISCUSSION TITLE
官网2.5.0版的场景案例的演示代码崩溃了

### DISCUSSION THREAD

**Discussion by emma-owen** [antvis/X6#3287] [Show and tell]
![image](https://user-images.githubusercontent.com/18736572/220851479-44432580-1bf3-41f3-8308-4a814d24aa28.png)

**Comment by Eve-Sama** [comment]
链接是这个吗 https://x6.antv.antgroup.com/zh/examples/showcase/practices/#dag
我这边打开是正常的

**Comment by NewByVector** [comment]
官网已经回滚，babel 新包有问题，https://github.com/babel/babel/issues/14469

**Comment by bodinsamuel** [comment]
Hello, just a heads up the examples are still broken, it seems the website hasn't been rollbacked yet/properly :)

**Reply by NewByVector** [reply]
Have been updated.

**Reply by bodinsamuel** [reply]
Thanks it works for me now ☺️
